packages/mongo-nav/src/MongoNav.tsx

```diff
@@ -237,6 +237,7 @@ function MongoNav({
           hosts={hosts}
           mode={mode}
           currentProjectName={data?.currentProject?.projectName}
+          currentProjectId={data?.currentProject?.projectId}
```

Comment:
We have `currentProjectId` on line 99 to use.

Reply:
👍